### ISSUE TITLE
Required groups vs Number of reviewers

### ISSUE THREAD

**Issue by albertdabrowski** [monitorjbl/pr-harmony#50]
We have 2 groups let's say A and B and want to say that we wanna have at least one PR approval from each group.
Currently if I have let's say 2 users in group A and get approvals from those 2 then it's enough.
Would it be possible to provide such logic so we require at least one approval from each required groups?

**Comment by ghost** [comment]
@albertdabrowski did you find a work around?

**Comment by albertdabrowski** [comment]
No. Due to this we decided to not use pr-harmony at all. So for us it is important to have at least one approval from one group (admins).
So we changed:
- branch permissions
- "default reviewers" feature in BB to only those with branch permissions and set required approvals to 1 from that group of people
So only person from that default reviewer may merge. Long way from ideal but enough for current needs.
